app/(main)/league/[leagueId]/entry/[entryId]/week/WeekHelper.tsx

```diff
@@ -25,25 +25,27 @@ import { IWeeklyPickChange } from './Week.interface';
  * @param props.week - Prop value for gameWeekId in updateWeeklyPicks
  * @param props.updateWeeklyPicks - Prop for the updateWeeklyPicks function
  * @param props.setUserPick - Prop for the setUserPick function
+ * @param props.setLoadingTeamName - Prop for the setLoadingTeamName state
  * @returns {void}
  */
 export const onWeeklyPickChange = async ({
   teamSelect,
   entry,
   league,
   NFLTeams,
+  setLoadingTeamName,
   setUserPick,
   updateWeeklyPicks,
   user,
   weeklyPicks,
   week,
 }: IWeeklyPickChange): Promise<void> => {
   try {
-    const teamID = NFLTeams.find(
-      (team) => team.teamName === teamSelect,
-    )?.teamName;
+    const team = NFLTeams.find((team) => team.teamName === teamSelect);
```

Comment:
Q: Why was this changed? I had to change it in one my tests and I'm curious if we had the same problem. Please let me know :)